### DISCUSSION TITLE
[Feature Request] 订阅通知，只通知给订阅用户

### DISCUSSION THREAD

**Discussion by heuyck** [jxxghp/MoviePilot#3455] [讨论]
### 当前程序版本

1.8.2

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

微信被通知的用户是否可以和MP的用户绑定？订阅通知只通知给订阅的用户

**Comment by heuyck** [comment]
也希望公共通知，例如站点统计、签到等等，能单独指定一个用户通知。不然通知频道里会比较乱

**Comment by behinder85** [comment]
同求，等待J大完成
